packages/scripts/src/monitoring-dvm2.0/common.ts

```diff
@@ -0,0 +1,61 @@
+import { VotingV2Ethers } from "@uma/contracts-node";
+import { BigNumber } from "ethers";
+import { increaseEvmTime } from "../utils/utils";
+
+const hre = require("hardhat");
+const { ethers } = hre;
+
+export const getUniqueVoters = async (votingV2: VotingV2Ethers): Promise<string[]> => {
+  const stakedEvents = await votingV2.queryFilter(votingV2.filters.Staked(null, null, null));
+
+  const uniqueVoters: string[] = [];
+  for (const event of stakedEvents) {
+    if (!uniqueVoters.includes(event.args.voter)) uniqueVoters.push(event.args.voter);
+  }
+  return uniqueVoters;
+};
+
+export const updateTrackers = async (votingV2: VotingV2Ethers, voters: string[]): Promise<void> => {
+  console.log("Updating trackers for all voters");
+  const tx = await votingV2.multicall(
+    voters.map((voter) => votingV2.interface.encodeFunctionData("updateTrackers", [voter]))
+  );
+  await tx.wait();
+  console.log("Done updating trackers for all voters");
+};
+
+export const getSumSlashedEvents = async (votingV2: VotingV2Ethers): Promise<BigNumber> => {
+  const voterSlashedEvents = await votingV2.queryFilter(votingV2.filters.VoterSlashed(), 0, "latest");
+  return voterSlashedEvents
+    .map((voterSlashedEvent) => voterSlashedEvent.args.slashedTokens)
+    .reduce((a, b) => a.add(b), ethers.BigNumber.from(0));
+};
+
+export const unstakeFromStakedAccount = async (votingV2: VotingV2Ethers, voter: string): Promise<void> => {
+  let stakeBalance = await votingV2.callStatic.getVoterStakePostUpdate(voter);
+
+  if (stakeBalance.gt(ethers.BigNumber.from(0))) {
+    console.log("Unstaking from", voter, stakeBalance.toString());
+    const impersonatedSigner = await ethers.getImpersonatedSigner(voter);
+    if ((await votingV2.voterStakes(voter)).pendingUnstake.gt(ethers.BigNumber.from(0))) {
+      console.log("Staker", voter, "has a pending unstake. Executing then re-unstaking");
+      await increaseEvmTime(await (await votingV2.unstakeCoolDown()).toNumber());
```

Comment:
`unstakeFromStakedAccount` is called for each unique voter so this could result unnecessary time increase. Alternative would be to check `unstakeRequestTime` for the voter and increase time only to reach end of cooldown.
Another approach would be to increase time in `unstake.ts` before iterating over all `uniqueVoters`.

Reply:
Fixed here: https://github.com/UMAprotocol/protocol/pull/4312/commits/c902b1a0aed15fba2b4d5db954bdd7b1888409f5